demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -809,7 +809,7 @@ def get_modified_and_added_files(self, compare_type, prev_ver):
             added_files = added_files - set(nc_modified_files) - set(nc_deleted_files)
             changed_meta_files = changed_meta_files - set(nc_deleted_files)
 
-        packs = self.get_packs(modified_files)
```

Comment:
maybe we should use this PR to change the name here to modified_packs just to clarify